cli/commands/makeSpend.ts

```diff
@@ -17,7 +17,7 @@ export class MakeSpend extends Command<CurrencyContext> {
     const { engine } = await getCliEngine(this.context)
 
     const spend: EdgeSpendInfo = {
-      tokenId,
+      tokenId: tokenId ?? null,
```

Comment:
Can we do `const {address, amount,tokenId = null} = this` up above? We don't want the `undefined` floating around - we should kill it right away.

Reply:
This one is still here.